codebreaker.js

```diff
@@ -275,19 +301,19 @@ function getMainMenuSelection(){
     let response = prompt("")
 
     switch(response){
-        case "i":{
+        case "i":
```

Comment:
It's unusual syntax, I know, but cases do *not* use curly braces to indicate a block of code.  That is why each case must end with a `break;`